src/doc/rustc/src/platform-support/riscv32imafc-unknown-none-elf.md

```diff
@@ -0,0 +1 @@
+riscv32imac-unknown-none-elf.md
```

Comment:
I did not know about this quirk of mdbook that turns it into a symlink. Cute.

Reply:
Hmm, wait, the table links to the same file... no, I think this is just not getting used at all...?

@MabezDev Was this intentional?

Reply:
Oops, you're right. I will fix that now.